playbooks/README.md

```diff
@@ -0,0 +1,21 @@
+# openshift-ansible playbooks
+
+In summary:
+
+- [`byo`](byo) (_Bring Your Own_ hosts) has the most actively maintained
+  playbooks for installing, upgrading and performing others tasks on OpenShift
+  clusters.
+
+And:
+
+- [`adhoc`](adhoc) is a generic home for playbooks and tasks, unsupported and
+  not maintained.
+- [`common`](common) was meant as a generic set of playbooks that would run
+  against different "providers" when the now deprecated
+  [`bin/cluster`](../bin/cluster) tool was supported.
```

Comment:
common is still in use, in that it backs the byo playbooks. Not sure how to best word that, but it is an important distinction